### DISCUSSION TITLE
Chrome in ubuntu-latest

### DISCUSSION THREAD

**Discussion by kkattamudi** [actions/virtual-environments#6165] [Installed software]
How do I get to know the path of the chrome and firefox browsers in ubuntu-latest image ?

**Comment by al-cheb** [comment]
@kkattamudi, hey.
You can use which command to get a path because those tools are in PATH:

```
- run: |
            which google-chrome
            which firefox
```

![image](https://user-images.githubusercontent.com/47745270/187378291-4c4f989b-28b4-43dc-8425-c72c9cd6d4ec.png)

**Reply by kkattamudi** [reply]
Can we open chrome browser with full user interface using selenium or wdio  using the this image ?

**Reply by al-cheb** [reply]
@kkattamudi , we don't preinstall a GUI on Ubuntu.

**Reply by kkattamudi** [reply]
Is there is any out of box ubuntu image with GUI  ? Or 

Is there is any out of box ubuntu image with GUI  + browsers ?

**Reply by al-cheb** [reply]
@kkattamudi, nope, if you need to use a GUI you should try installing it in runtime or use a self-hosted runner.

**Comment by samus-aran** [comment]
We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
